tests/test_notifications.py

```diff
@@ -401,16 +400,17 @@ def test_get_all_user_subscriptions(self):
         assert_in(self.project_subscription, user_subscriptions)
         assert_in(self.node_subscription, user_subscriptions)
         assert_in(self.user_subscription, user_subscriptions)
-        assert_equal(len(user_subscriptions), 3)
+        assert_equal(len(user_subscriptions), 5)
 
     def test_get_all_node_subscriptions_given_user_subscriptions(self):
         user_subscriptions = utils.get_all_user_subscriptions(self.user)
         node_subscriptions = [x for x in utils.get_all_node_subscriptions(self.user, self.node,
-                                                                          user_subscriptions=user_subscriptions)]
+                                                                          user_subscriptions=user_subscriptions)
+                                if x.event_name != 'mailing_list_events']
```

Comment:
Why are we filtering out `mailing_list_events` here? Why aren't those included in `get_all_user_subscriptions`?